packages/contact-center/user-state/tests/user-state/index.tsx

```diff
@@ -27,6 +25,10 @@ describe('UserState Component', () => {
     };
     global.Worker = jest.fn(() => workerMock);
     global.URL.createObjectURL = jest.fn(() => 'blob:http://localhost:3000/12345');
+
+    if (typeof window.HTMLElement.prototype.attachInternals !== 'function') {
+      window.HTMLElement.prototype.attachInternals = jest.fn() as any;
+    }
```

Comment:
This is similar to how we mocked the web components earlier. We do not need to test any part of the UI lib here as we do that in the `cc-components` tests anyway